discovery/wh_discovery.go

```diff
@@ -147,12 +147,9 @@ func deserializeWebhookJSON(body []byte) ([]common.OrchestratorLocalInfo, error)
 	}
 	var infos []common.OrchestratorLocalInfo
 	for _, addr := range addrs {
-		if addr.Address == "" {
```

Comment:
We probably want to keep this in for resiliency, since the JSON parser will still deserialize objects with empty addresses (unless `ParseRequestURI` returns an error on empty inputs; can you confirm that?)